indico/modules/events/themes.yaml

```diff
@@ -0,0 +1,298 @@
+defaults:
+    conference: static
+    simple_event: lecture
+    lecture: lecture
+    meeting: standard
+definitions:
+  # Lectures - base
+  event:
+    event_types:
+    - simple_event
```

Comment:
I'd move those to the previous line, e.g. `event_types: [simple_event]`, or at least indent them